host/tests/L3_TestCases/dsFPD/dsFPD_test02_SetVerifyLEDIndicatorsBrightness.py

```diff
@@ -130,5 +130,7 @@ def testFunction(self):
         return result
 
 if __name__ == '__main__':
+    summerLogName = os.path.splitext(os.path.basename(__file__))[0] + "_summery"
+    summeryLog = logModule(summerLogName, level=logModule.INFO)
     test = dsFPD_test02_SetVerifyLEDIndicatorsBrightness()
```

Comment:
summeryLog variable is not used anywhere.. it should be passed in line 135

Reply:
Updated to pass summaryLog object to test object.